src/Events/Integrations/Plugins/Tickets_Wallet_Plus/Passes/Apple_Wallet/Event_Modifier.php

```diff
@@ -13,10 +15,26 @@
  */
 class Event_Modifier {
 
+	/**
+	 * Format for displaying the Date
+	 *
+	 * @var string
+	 */
+	protected $date_format = 'M j';
+
+	/**
+	 * Format for displaying the time.
+	 *
+	 * @var string
+	 */
+	protected $time_format = 'g:ia';
+
+
```

Comment:
Missing `@since` Shouldn't these come from the settings we already have for formats?

Reply:
I asked about this already. I think we are using these because the design specified a certain format. If we get support requests about it, we may want to rethink it.

Reply:
Howdy, the dates are not to print on the site FE but on the Apple Wallet Pass, which is different. 

We should have a filter for these formats (Like, in some countries, they may want to have 17:00h and some others 5:00PM). 

But the formats are particular to the design of the Apple Wallet pass, and not linked to WordPress dates (the formats in the settings).